job_executor/model/metadata.py

```diff
@@ -188,6 +196,11 @@ def remove_none(cls, values):
             if value is not None
         }
 
+    @staticmethod
+    def data_type_to_ssb_model(data_type: str):
```

Comment:
Any reason we are using this method rather than looking up in the module dictionary directly?

Reply:
No reason :) I fix